code/game/objects/items/implants/implant_bb.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/implant/bloodbrother
+	name = "communication implant"
+	desc = "Use this to communicate with your fellow blood brother(s)."
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "headset"
+	item_color = "r"
+	var/list/linked_implants // All other implants that this communicates to
+
+/obj/item/implant/bloodbrother/Initialize()
+	. = ..()
+	linked_implants = list()
+
+/obj/item/implant/bloodbrother/activate()
+	. = ..()
+	if(linked_implants.len)
+		var/input = stripped_input(imp_in, "Enter a message to communicate to your blood brother(s).", "Radio Implant", "")
```

Comment:
Also add a conscious check after it, since otherwise you can hold the menu open until you die then just send 'I died' afterwards and communicate from the spirit realm